tests/cnf/core/network/metallb/tests/bgp-remote-as-dynamic.go

```diff
@@ -179,3 +250,79 @@ func setupBGPRemoteASTestCase(hubIPv4ExternalAddresses, externalAdvertisedIPv4Ro
 
 	return frrk8sPods, frrPod
 }
+
+func setupBGPRemoteASMultiHopTest(ipv4metalLbIPList, hubIPv4ExternalAddresses, externalAdvertisedIPv4Routes,
+	externalAdvertisedIPv6Routes, hubPodWorkerName []string, frrExternalMasterIPAddress string, asNumber int,
+	eBGP bool) (*pod.Builder, []*pod.Builder) {
+	var (
+		nodeAddrList []string
+	)
+
+	By("Creating a new instance of MetalLB Speakers on workers")
+
+	err := metallbenv.CreateNewMetalLbDaemonSetAndWaitUntilItsRunning(tsparams.DefaultTimeout, workerLabelMap)
+	Expect(err).ToNot(HaveOccurred(), "Failed to recreate metalLb daemonset")
+
+	By("Collecting information before test")
+
+	frrk8sPods, err := pod.List(APIClient, NetConfig.MlbOperatorNamespace, metav1.ListOptions{
+		LabelSelector: tsparams.FRRK8sNodeLabel,
+	})
+	Expect(err).ToNot(HaveOccurred(), "Failed to list speaker pods")
+	By("Setting test iteration parameters")
```

Comment:
Not sure, is it a correct. You don't have any itterations, right?

Reply:
changed to "Setting test parameters"